config/global.ini.php

```diff
@@ -45,6 +45,10 @@
 ; Matomo should work correctly without this setting but we recommend to have a charset set.
 charset = utf8
 
+; Matomo recommend enabling this setting if the default charset is set to utf8mb4
```

Comment:
can we slightly tweak this to `Matomo recommends enabling this setting if the default charset is set to "utf8mb4". It can avoid the MySQL error "row size too large" for new tables.`?